Lekce_05_BR01/Program.cs

```diff
@@ -131,9 +128,15 @@ public static void Main()
         };
 
             // 7. Řešení
-            // najít milionáře a pak dát grouping podle kodu banky
 
-            List<SkupinaMilionaru> skupinyPodleBanky = null;
+            List<SkupinaMilionaru> skupinyPodleBanky = zakaznici
+                .Where(x => x.Zustatek >= 1000000)
+                .GroupBy(x => x.Banka, (kodBanky, milionari) => new SkupinaMilionaru
+                {
+                    Banka = kodBanky,
+                    Milionari = milionari.Select(x => x.Jmeno)
+                })
```

Comment:
nit: stejne tak tady :) 
```suggestion
                .GroupBy(z => z.Banka, (kodBanky, milionari) => new SkupinaMilionaru
                {
                    Banka = kodBanky,
                    Milionari = milionari.Select(m => m.Jmeno)
                })
```